src/components/Search/SearchContainer.js

```diff
@@ -20,6 +20,7 @@ export const SearchContainer = ({
       className={cx(styles.wrapper, className)}
       iconPosition='left'
       onSuggestionSelect={({ slug }) => history.push(`/projects/${slug}`)}
+      sorter={({ marketcapUsd: a }, { marketcapUsd: b }) => b - a}
```

Comment:
let's use `rank` instead `marketcapUsd` (we will have that field in our json, as I understand)

Reply:
@Partysun , @haritonasty here are using such query
![image](https://user-images.githubusercontent.com/14061779/67393710-946e9a80-f5ab-11e9-9e57-ac9f344ed69f.png)
I will add a rank for it

Reply:
done